roles/contiv_network/tasks/ovs.yml

```diff
@@ -13,6 +13,10 @@
   tags:
     - prebake-for-dev
 
+- name: install openvswitch (Redhat)
```

Comment:
This task might have come from merge. ovs is installed at line 10 above, so this task can be removed.